src/eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -1262,7 +1276,9 @@ def save_fit(meta, lc, model, fitter, results_table, freenames, samples=[]):
 
     # Save the S5 outputs in a human readable ecsv file
     event_ap_bg = meta.eventlabel+"_ap"+str(meta.spec_hw)+'_bg'+str(meta.bg_hw)
-    if lc.share:
+    if lc.white:
```

Comment:
We should define channel_tag sooner and then make fname simpler: `fname = f'S5_{fitter}_samples{channel_tag}'.